.travis.yml

```diff
@@ -5,8 +5,8 @@ language: scala
 sudo: false
 
 scala:
-  - "2.12.11"
-  - "2.13.3"
+  - "2.12.14"
+  - "2.13.6"
```

Comment:
I see the travis builds use `akka.build.scalaVersion` which ignores the patch version anyway. Maybe a comment clarifying that would be enough rather than updating the version here each time?